src/test/resources/com/puppycrawl/tools/checkstyle/checks/design/designforextension/InputDesignForExtensionOverridableMethods.java

```diff
@@ -147,7 +147,7 @@ public int foo30() {
             return 1;
         }
 
-        /* */
+        /* */ // violation
```

Comment:
Please add test where emplty javadoc and non empty block comment a next to each other.
Javadoc above block comment.
Javadoc below block comment.

Reply:
Done.